### PR TITLE
Add guide to connect Google Workspace with OVHcloud account

### PR DESCRIPTION
**Pull request by VincentCasse**
Hey,

Here a documentation to help our customer to connect their own Google Workspace and their OVHcloud account.
I hope this will follow your rules. Please give me feedbacks

### CODE REVIEW

File: pages/account/customer/ovhcloud-account-connect-google-workspace/guide.en-us.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+section: Utilisation avancée
+order: 02
+updated: 2023-03-24
+---
+
+**Dernière mise à jour le 24-03-2023**
+
+## Objectif
+
+Vous pouvez utiliser l'authentification SSO (*Single Sign-On*) **unique** pour vous connecter à votre compte OVHcloud. Pour activer ces connexions, votre compte et vos comptes Google Workspace doivent être configurés à l'aide des authentifications SAML (*Security Assertion Markup Language*).
```
Comment (sebferrer) [inline pages/account/customer/ovhcloud-account-connect-google-workspace/guide.en-us.md]: "votre compte et vos comptes" --> tu voulais dire votre compte OVHcloud?

File: pages/account/customer/ovhcloud-account-connect-google-workspace/guide.en-us.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+
+Votre Google Workspace est maintenant considéré comme fournisseur d'identité de confiance. Cependant, vous devez tout de même ajouter des groupes à votre compte OVHcloud.
+
+> [!warning]
+> Si vous essayez à ce stade de vous connecter via SSO, un message d'erreur `Not in valid groups` s'affichera probablement.
+>
+> En effet, votre compte OVHcloud vérifie si l'utilisateur s'authentifiant appartient à un groupe existant sur le compte.
+>
+
+Pour cela, vous devez autoriser les groupes qui seront transmis de Google Workspace vers OVHcloud. Ces groupes sontles même que ceux utilisés pour catégoriser vos utilisateurs.
```
Comment (sebferrer) [inline pages/account/customer/ovhcloud-account-connect-google-workspace/guide.en-us.md]:
```suggestion
Pour cela, vous devez autoriser les groupes qui seront transmis de Google Workspace vers OVHcloud. Ces groupes sont les mêmes que ceux utilisés pour catégoriser vos utilisateurs.
```

**Review by sebferrer — DISMISSED** [review]

File: pages/account/customer/ovhcloud-account-connect-google-workspace/guide.fr-fr.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+
+Votre Google Workspace est maintenant considéré comme fournisseur d'identité de confiance. Cependant, vous devez tout de même ajouter des groupes à votre compte OVHcloud.
+
+> [!warning]
+> Si vous essayez à ce stade de vous connecter via SSO, un message d'erreur `Not in valid groups` s'affichera probablement.
+>
+> En effet, votre compte OVHcloud vérifie si l'utilisateur s'authentifiant appartient à un groupe existant sur le compte.
+>
+
+Pour cela, vous devez autoriser les groupes qui seront transmis de Google Workspace vers OVHcloud. Ces groupes sont les même que ceux utilisés pour catégoriser vos utilisateurs.
```
Comment (OVH-Team-Guides) [inline pages/account/customer/ovhcloud-account-connect-google-workspace/guide.fr-fr.md]:
```suggestion
Pour cela, vous devez autoriser les groupes qui seront transmis de Google Workspace vers OVHcloud. Ces groupes sont les mêmes que ceux utilisés pour catégoriser vos utilisateurs.
```

**Review by OVH-Team-Guides — APPROVED** [review]

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @VincentCasse

**Pull request merged**